### PR TITLE
Add Arduino books in pt-BR

### PR DESCRIPTION
**Pull request by Samuel-de-Oliveira**
I saw it's was missing books about Arduino in Brazilian Portuguese language, so i decided to add them.

### CODE REVIEW

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -82,6 +83,11 @@
 
 * [Google Android: Uma abordagem prática e didática](https://leanpub.com/google-android) - Rafael Guimarães Sakurai - (Necessário criar uma conta (gratuita) no Leanpub para baixar o livro completo nos formatos PDF, EPUB, MOBI ou pelo próprio site)
 
+### Arduino
+
+* [Apostilas da Eltrogate](https://www.eletrogate.com/pagina/apostilas.html) - Eletrogate (No site é possivel escolher entre 9 apostilas diferentes dependendo do seu nível, e será pedido seu email para a obtenção do documento, todas elas são em formato PDF)
+* [Arduino Guia Iniciante](https://edisciplinas.usp.br/pluginfile.php/3252633/mod_resource/content/1/Guia_Arduino_Iniciante_Multilogica_Shop.pdf) - Multilógica Shop (PDF)
+* [Física com Arduino para iniciantes](https://www.scielo.br/j/rbef/a/rrPFbrjsDdtkKKRPDPTJXnn/?format=pdf&lang=pt) - Marisa Almeida Cavalcante, Cristiane Rodrigues Caetano Tavolaro, Elio Molisani (PDF)
```
Comment (LuigiImVector) [inline books/free-programming-books-pt_BR.md:91]:
```suggestion


### Arduino

* [Apostilas da Eltrogate](https://www.eletrogate.com/pagina/apostilas.html) - Eletrogate (No site é possivel escolher entre 9 apostilas diferentes dependendo do seu nível, e será pedido seu email para a obtenção do documento, todas elas são em formato PDF)
* [Arduino Guia Iniciante](https://edisciplinas.usp.br/pluginfile.php/3252633/mod_resource/content/1/Guia_Arduino_Iniciante_Multilogica_Shop.pdf) - Multilógica Shop (PDF)
* [Física com Arduino para iniciantes](https://www.scielo.br/j/rbef/a/rrPFbrjsDdtkKKRPDPTJXnn/?format=pdf&lang=pt) - Marisa Almeida Cavalcante, Cristiane Rodrigues Caetano Tavolaro, Elio Molisani (PDF)


```

You didn't respect the [formatting guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#formatting), two blank lines were missed.

Next time fill in the default PR template.

File: books/free-programming-books-pt_BR.md [hunk 2]
```diff
@@ -83,6 +84,13 @@
 * [Google Android: Uma abordagem prática e didática](https://leanpub.com/google-android) - Rafael Guimarães Sakurai - (Necessário criar uma conta (gratuita) no Leanpub para baixar o livro completo nos formatos PDF, EPUB, MOBI ou pelo próprio site)
 
 
+### Arduino
+
+* [30 Projetos com Arduino](https://www.academia.edu/36112333/30_Projetos_com_Arduino_pdf) - Simon Monk (É necessario uma conta na Academia.edu para baixar o PDF)
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:89]: We will accept links to Academia.edu only if they have been posted there by the author. So we won't link to this one.

**Comment by Samuel-de-Oliveira** [comment]
I don't found the original source of the book, so I decided to remove.

P. S.  the first commit I understood wrong, so I fixed it.

File: books/free-programming-books-pt_BR.md [hunk 3]
```diff
@@ -83,6 +84,12 @@
 * [Google Android: Uma abordagem prática e didática](https://leanpub.com/google-android) - Rafael Guimarães Sakurai - (Necessário criar uma conta (gratuita) no Leanpub para baixar o livro completo nos formatos PDF, EPUB, MOBI ou pelo próprio site)
 
 
+### Arduino
+
+* [Arduino Guia Iniciante](https://edisciplinas.usp.br/pluginfile.php/3252633/mod_resource/content/1/Guia_Arduino_Iniciante_Multilogica_Shop.pdf) - Multilógica Shop (PDF)
+* [Física com Arduino para iniciantes](https://www.scielo.br/j/rbef/a/rrPFbrjsDdtkKKRPDPTJXnn/?format=pdf&lang=pt) - Marisa Almeida Cavalcante, Cristiane Rodrigues Caetano Tavolaro, Elio Molisani (PDF)
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:90]: This is a 9 page article published in i journal. It's not a book.

**Review by eshellman — CHANGES_REQUESTED** [review]
When submitting a PR, please answer the questions in the form!

**Review by davorpa — APPROVED** [review]
LGTM. Thanks for changes

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**